crates/forge/src/collecting.rs

```diff
@@ -130,6 +142,15 @@ pub(crate) fn filter_tests_from_crates(
             .collect()
     } else {
         compiled_test_crates
+    };
+
+    if let TestsToRun::Ignored = runner_config.tests_to_run {
+        filtered_by_name
+            .into_iter()
+            .map(CompiledTestCrate::extract_ignored_cases)
+            .collect()
```

Comment:
Representing it as a filter primitive would be much more intuitive and consistent